### PR TITLE
refactor(file-structure): change the file structure

### LINKED ISSUE #11: File architecture and restructuring
**Is your feature request related to a problem? Please describe.**
Restructuring the files and format

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Created and moved files and folders

## Purpose
To enable us to work independently on the application

Closes #11

### CODE REVIEW

File: movie-app/src/App.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import "./App.css";
 import React, { useState, useEffect } from "react";
-import MovieCard from "./Components/MovieCard";
-import MovieDetails from "./Components/MovieDetails";
+import MovieCard from "./components/MovieCard";
+import MovieDetails from "./components/MovieDetails";
```
Comment (BlackCubes) [inline movie-app/src/App.js:4]: Nice file structure and naming conventions!

File: movie-app/src/header/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const NavBar = () => {
+    return(
+        <div>
+            {/* Logo */}
+            {/* Search Bar */}
+            {/* Hero */}
```
Comment (BlackCubes) [inline movie-app/src/header/NavBar.js:6]: I like the the comments on what your team is going to put!

**Review by BlackCubes — APPROVED** [review]
Everything looks simple and well thought out. Good job! Can't wait to see it 👍

**Review by JayDampitan — APPROVED** [review]
Ya'll doing great!

**Review by dannysanchez559 — APPROVED** [review]
I'd buy it!

**Pull request merged**